flutter/lib/common.dart

```diff
@@ -196,6 +183,8 @@ class MyTheme {
     ),
   );
 
+  static const SwitchThemeData switchTheme = SwitchThemeData(splashRadius: 0);
```

Comment:
No switch used. SwitchTheme can be removed?

Reply:
Switches are used at android server/settings page, they might affected by this change. not sure.